.changeset/cuddly-pugs-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@datadog/create-app': major
```

Comment:
Bumping the other packages due to peer dependencies updates.

```suggestion
'@datadog/create-app': major
'@datadog/ui-extensions-react': minor
'@datadog/ui-extensions-sdk': minor
```

Again, feel free to ignore this, or make a separate changeset, if you'd prefer.

Reply:
@joneshf-dd Good catch! After going down the dependency rabbit hole I totally forgot to add a changeset for the other packages I had to touch.

I'll get that updated and pushed out.